server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/GroupWsRef.java

```diff
@@ -69,7 +71,7 @@ public String getOrganizationUuid() {
   }
 
   /**
-   * @return the non-null group name.
+   * @return the non-null group name. Can be anyone.
```

Comment:
best would be to reference a constant which value is `anyone` (is it this exact case?). Would allow client code to use this constant to check the returned value if need be and would keep documentation up to date with code most easily